learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Analytics
+description: Meilisearch Cloud offers in-depth search analytics to help you understand how users search in your application.
+sidebarDepth: 3
+---
+
+# Analytics
+
+Enable Meilisearch Cloud analytics to gain insight on what your users are searching for, how often they find it, and how much they have to wait between typing a query and receiving results.
+
+## Requirements
+
+You must have a Meilisearch Cloud account to have access to search analytics.
+
+## Activating analytics
+
+### Enable analytics in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics" section and click on the "Enable analytics" button:
+
+![The analytics section of the project overview. It shows one button, "Enable analytics", and a short explanation of the feature.](https://raw.githubusercontent.com/meilisearch/documentation/cloud-analytics-guide/assets/images/cloud-analytics/01-analytics-enable.png)
+
+Meilisearch Cloud will begin processing your request. The "Analytics" section will update when the feature is enabled.
+
+### Update URLs in your application
+
+Enabling analytics generates a new URL for your project. Meilisearch Cloud is only able to track metrics for queries sent to this URL. 
```

Comment:
Qogita has reported that "a new URL for your project" can be confusing. Should we say that it's a new host for the meilisearch instance or something like this?

Reply:
If we are very strict, the naming URL is correct, but I can understand why they misunderstood. What about something like this:

Enabling analytics requires an update in your Meilisearch integration's implementation. Meilisearch Cloud is only able to track metrics for queries sent to this URL. Instead of using the old URL like `ms-xxxxxx-123.sfo.meilisearch.com`, for example, you'll have to switch to `edge.meilisearch.com`.

Reply:
Just for the record, we got [another support request](https://app.crisp.chat/website/38a71dd2-729c-4970-b061-3e1db1c1eb20/inbox/session_a22624a3-1f38-4856-93c0-48e3e2459212/) asking

> I've noticed that the hostname on the dashboard is currently set as edge.meilisearch.com. Should I consider changing it from the previous format, which was similar to ms-2918e39a832a-2300.sgp.meilisearch.io? Is there a specific advantage or requirement for switching to the new format?